src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -7,8 +7,12 @@
 )
 
 from deprecated import deprecated
+from furl import furl
 
 from azul import config
+from azul.dss import (
+    shared_credentials,
+)
```

Comment:
```suggestion
from azul.dss import shared_credentials
```

Reply:
Not implementing this as the upcoming import enforcement PR with make the current syntax a requirement.